### ISSUE TITLE
`upstream` CI issue creation error 

### ISSUE THREAD

**Issue by jrbourbeau** [dask/dask#9681]
Looking at [our latest `upstream` build](https://github.com/dask/dask/actions/runs/3518074457/jobs/5896586141), it appears we're getting this error when we attempt to create / update the issue corresponding to our `upstream` CI 

```
Run python $GITHUB_ACTION_PATH/parse_logs.py output-log.jsonl
  python $GITHUB_ACTION_PATH/parse_logs.py output-log.jsonl
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    COVERAGE: true
    INPUT_RUN_POST: true
    CONDA: /usr/share/miniconda3
    CONDA_PKGS_DIR: /home/runner/conda_pkgs_dir
Traceback (most recent call last):
  File "/home/runner/work/_actions/xarray-contrib/issue-from-pytest-log/v1.2/parse_logs.py", line 39, in <module>
    class PreformattedReport:
  File "/home/runner/work/_actions/xarray-contrib/issue-from-pytest-log/v1.2/parse_logs.py", line 42, in PreformattedReport
    variant: str | None
TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
```

cc @keewis for visibility -- have you seen this before?

**Comment by keewis** [comment]
yes, the reason seems to be that the action did not specify `bash -l {0}` as the shell. This is fixed in `v1.2.3`, so you can either upgrade to that or use the rolling `v1`  tag.

**Comment by jrbourbeau** [comment]
Ah, good to know. Addressing in https://github.com/dask/dask/pull/9682. Thanks @keewis!
